mmrazor/models/pruners/structure_pruning.py

```diff
@@ -844,8 +853,7 @@ def concat_backward_parser(self, grad_fn, module2name, var2module,
         # However, a shared module will be visited more than once during
         # forward, so it is still need to be traced even if it has been
         # visited.
-        if (name in visited and visited[name]
-                and name not in self.shared_module):
+        if name in visited and visited[name] and name not in self.shared_module:
```

Comment:
This line is too long to pass the lint check.
More details can be found at https://github.com/open-mmlab/mmrazor/runs/7088305507?check_suite_focus=true